source/includes/reference/_profiles.md

```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
 issuerState              | Issued by state code. For example "NY"      | Text
 expiryDate               | Document expiry date. (optional)            | YYYY-MM-DD
 
+
+## Get business directors
+
+> Example Request:
+
+```shell
+curl -X GET https://api.sandbox.transferwise.tech/v1/profiles/{profileId}/directors \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+[
+  {
+    "id": <director id>,
+    "firstName": "John",
+    "lastName": "Doe",
+    "dateOfBirth": "1982-05-20",
```

Comment:
Make this more obviously fake data - e.g. `01-01`, security ops told me before to make sure our example data is obviously fake.

Reply:
Done